src/test/scala/nl/knaw/dans/easy/schemaExamples/SchemaValidationFixture.scala

```diff
@@ -15,19 +15,128 @@
+  protected val httpsEasySchemaBase = s"https:$publicEasySchemaBase"
+
+  private lazy val triedPublicSchema: Try[Schema] = Try {
+    SchemaFactory
+      .newInstance(XMLConstants.W3C_XML_SCHEMA_NS_URI)
+      .newSchema(Array(new StreamSource(publicSchema.toString)).toArray[Source])
+  }
+  private lazy val triedLocalSchema: Try[Schema] = Try {
+    // lazy for two reasons:
+    // - schemaFile is set by concrete test class
+    // - postpone loading until actually validating
+    val xsdInputStream = File(localSchemaFile).contentAsString.replaceAll(
+      s"""schemaLocation="http:$publicEasySchemaBase""",
+      s"""schemaLocation="file://$schemaDir"""
+    ).inputStream
+    SchemaFactory
+      .newInstance(XMLConstants.W3C_XML_SCHEMA_NS_URI)
+      .newSchema(Array(new StreamSource(xsdInputStream)).toArray[Source])
+  }
+
+  "examples" should "be schema valid with local copy of easy-schema" in {
+    forEvery(examples) { example =>
+      val xml = XML.loadFile((exampleDir / example).toString())
+      easyLocationsIn(xml).foreach(File(_).toJava should exist)
+      assume(schemaIsOnline(triedLocalSchema)) // inside the loop to not block other errors
+      validate(triedLocalSchema, toLocalSchemas(xml)) shouldBe a[Success[_]]
+    }
+  }
+
+  it should "be schema valid with (unqualified) public schema" in {
+    // sub-schemas may not have an unqualified public schema
+    assume(schemaIsOnline(triedPublicSchema)) // outside the loop as we are only validating
+    assume(lastLocalIsPublic)
+    forEvery(examples) { example =>
+      val xml = XML.loadFile((exampleDir / example).toString())
+      validate(triedPublicSchema, xml.toString) shouldBe a[Success[_]]
+    }
+  }
+
+  private def lastLocalIsPublic: Boolean = {
+    // to ignore tests while easy-schema and easy-schema-examples are under parallel development
+    // and the new schema's are not yet published
+    // reported with: ...lastLocalIsPublic was false
+    val newPublicSchema = localSchemaFile.replace(schemaDir.toString(), httpsEasySchemaBase)
+    Try(new URL(newPublicSchema).openStream()).fold(_ => false, _ => true)
+  }
+
+  private def validate(schema: Try[Schema], xmlString: String): Try[Unit] = {
+    schema.map(_.newValidator().validate(new StreamSource(xmlString.inputStream))) match {
+      case Failure(e: SAXParseException) =>
+        showErrorWithSourceContext(xmlString, e)
+        Failure(e)
+      case x => x
+    }
+  }
+
+  private def easyLocationsIn(xml: Elem): Seq[String] = {
+
+    val attributeValues = xml.attributes.asAttrMap.map {
+      case ("xsi:noNamespaceSchemaLocation", value: String) => Some(s"dummyUri $value")
+      case ("xsi:schemaLocation", value: String) => Some(value)
+      case _ => None
+    }.filter(_.isDefined).flatten
+    // now we should have strings formatted like: "nsUri1 xsdUrl1 nsUri2 xsdUrl2"
+    attributeValues
+      .flatMap(_.trim.split(" +").grouped(2).toList)  // extract (uri,url) tuples
```

Comment:
`.toList` isn't necessary

```suggestion
      .flatMap(_.trim.split(" +").grouped(2))  // extract (uri,url) tuples
```